master.tf

```diff
@@ -23,29 +23,37 @@ resource "linode_instance" "k8s_master" {
     kernel = "linode/direct-disk"
 
     devices {
-      sda = {
+      sda {
         disk_label = "boot"
       }
     }
   }
 
   provisioner "file" {
-    source      = "${path.module}/scripts/"
+    source      = "${path.cwd}/${path.module}/scripts/"
     destination = "/tmp"
 
     connection {
-      user    = "core"
-      timeout = "300s"
+      host        = self.ip_address
+      agent       = "false"
+      private_key = chomp(file(var.ssh_private_key))
+      type        = "ssh"
+      user        = "core"
+      timeout     = "300s"
     }
   }
 
   provisioner "file" {
-    source      = "${path.module}/manifests-tmp/"
+    source      = "${path.cwd}/${path.module}/manifests-tmp/"
     destination = "/tmp"
 
     connection {
-      user    = "core"
-      timeout = "300s"
+      host        = self.ip_address
+      agent       = "false"
```

Comment:
Same comments as above.

Reply:
I switched agent = "false" to agent = "true" for all instances. 

The host line is required.